setup.py

```diff
@@ -484,26 +487,35 @@ def cmake_available():
     Checks whether CMake command is available and >= version 3.3.
     :return:
     """
+    CMAKE_MINIMUM_MAJOR = 3
+    CMAKE_MINIMUM_MINOR = 3
     try:
         output = subprocess.check_output(["cmake", "--version"]).split()
         version = output[2].decode("utf-8").split(".")
-        return int(version[0]) >= 3 and int(version[1]) >= 3
+        return (
+            int(version[0]) >= CMAKE_MINIMUM_MAJOR
+            and int(version[1]) >= CMAKE_MINIMUM_MINOR
+        )
     except:
         return False
 
 
+def parse_requirements(req_file):
+    with open(req_file) as f:
+        return f.read().strip().split("\n")
```

Comment:
`str.splitlines()`?

Reply:
That will leave entries for any newline(s) at the end of the file.